# Can one block of except statements handle multiple exception?

# this are answer of the above theory question
k="Yes A Single Block of Except Statements handle multiple Exception ."
print(k)